lms/djangoapps/certificates/views/webview.py

```diff
@@ -254,7 +254,22 @@ def _update_course_context(request, context, course, platform_name):
     course_number = course.display_coursenumber if course.display_coursenumber else course.number
     context['course_number'] = course_number
     context['is_integrity_signature_enabled_for_course'] = is_integrity_signature_enabled(course.location.course_key)
-    context['accomplishment_copy_course_description'] = context['certificate_data'].get('course_description', '')
+    if context['certificate_data'].get('course_description', ''):
+        context['accomplishment_copy_course_description'] = context['certificate_data']['course_description']
+    elif context['organization_long_name']:
+        # Translators:  This text represents the description of course
+        context['accomplishment_copy_course_description'] = _('a course of study offered by {partner_short_name}, '
+                                                              'an online learning initiative of '
+                                                              '{partner_long_name}.').format(
+            partner_short_name=context['organization_short_name'],
+            partner_long_name=context['organization_long_name'],
+            platform_name=platform_name)
+    else:
+        # Translators:  This text represents the description of course
+        context['accomplishment_copy_course_description'] = _('a course of study offered by '
+                                                              '{partner_short_name}.').format(
+            partner_short_name=context['organization_short_name'],
+            platform_name=platform_name)
```

Comment:
Thank you for the changes @pkulkark, just a small nit, feel free to ignore it. It may not help the formatting a lot at all. How about this?

```py
course_description_override = context['certificate_data'].get('course_description')

if course_description_override:
    accomplishment_copy_course_description = course_description_override
elif context['organization_long_name']:
    # Translators:  This text represents the description of course
    accomplishment_copy_course_description = _(
        'a course of study offered by {partner_short_name}, '
        'an online learning initiative of '
        '{partner_long_name}.'
    ).format(
        partner_short_name=context['organization_short_name'],
        partner_long_name=context['organization_long_name'],
        platform_name=platform_name,
    )
else:
    # Translators:  This text represents the description of course
    accomplishment_copy_course_description = _('a course of study offered by {partner_short_name}.').format(
        partner_short_name=context['organization_short_name'],
        platform_name=platform_name,
    )


context['accomplishment_copy_course_description'] = accomplishment_copy_course_description
```

Reply:
Added it